source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRToggleVisibilityTarget.mm

```diff
@@ -38,32 +39,49 @@ - (void)doSelectAction
         NSString *hashString = [NSString stringWithCString:target->GetElementId().c_str() encoding:NSUTF8StringEncoding];
         NSUInteger tag = hashString.hash;
         UIView *view = [_rootView viewWithTag:tag];
+        UIView *separator = nil;
+        BOOL bHide = NO;
```

Comment:
what does the b stand for? if the variable holds the visibility value that the element will have could it be a better idea to name the variable with something related to visible?

Reply:
nevermind, the whole concept of the visibility manager uses the word 'hide' or 'hidden' so it's ok to keep it under those terms